testbed/tests/test_app.py

```diff
@@ -313,8 +313,12 @@ async def test_current_window(app, app_probe, main_window):
         """The current window can be retrieved."""
         try:
             assert app.current_window == main_window
+
+            # When all windows are hidden, WinForms and Cocoa return None, while GTK
+            # returns the last active window.
             main_window.hide()
-            assert app.current_window is None
+            assert app.current_window in [None, main_window]
```

Comment:
We can definitely live with this for now, but it would be worth logging as a bug that *should* be possible to fix.